wasm/build.js

```diff
@@ -19,21 +20,28 @@ async function buildRollup(input, output) {
 async function buildWasm(network) {
     await buildRollup({
         input: {
-            "aleo_wasm": "entry",
+            "aleo_wasm": "./Cargo.toml",
+            "aleo_wasm_custom": "./Cargo.toml?custom",
         },
         plugins: [
-            virtual({
-                "entry": `export { default } from "./Cargo.toml";`,
-            }),
-
             rust({
-                cargoArgs: [
-                    // This enables multi-threading
-                    "--config", `build.rustflags=["-C", "target-feature=+atomics,+bulk-memory,+mutable-globals", "-C", "link-arg=--max-memory=4294967296"]`,
-                    "--no-default-features",
-                    "--features", `browser,${network}`,
-                    "-Z", "build-std=panic_abort,std",
-                ],
+                extraArgs: {
+                    cargo: [
+                        "--config", "build.rustflags=" + JSON.stringify([
+                            // This enables multi-threading
+                            "-C", "target-feature=+atomics,+bulk-memory,+mutable-globals",
+                            "-C", "link-arg=--max-memory=4294967296",
+
+                            // Strips out debug information
+                            "-Z", "location-detail=none",
+                            "-Z", "fmt-debug=none",
+                        ]),
+
+                        "--no-default-features",
+                        "--features", `browser,${network}`,
+                    ],
+                    wasmOpt: ["-O", "--enable-threads", "--enable-bulk-memory", "--enable-bulk-memory-opt"],
```

Comment:
If I remember correctly, you've rebuilt the interface in the rollup between Rayon and the web workers correct?

Reply:
This doesn't touch the Worker/Rayon stuff, it's just upgrading to the latest version of `@wasm-tool/rollup-plugin-rust`.

There was a bug that was preventing wasm-opt from running, this PR fixes that, which is why it needs special wasm-opt arguments.